clinica/iotools/bids_utils.py

```diff
@@ -331,10 +332,10 @@ def create_scans_dict(
         for session_id in {"ses-" + key for key in ses_dict[bids_id].keys()}:
             scans_dict[bids_id][session_id] = {
                 "T1/DWI/fMRI/FMAP": {},
-                "PIB": {},
-                "AV45": {},
-                "FLUTE": {},
-                "FDG": {},
+                Tracer.PIB: {},
+                Tracer.AV45: {},
```

Comment:
Given that we are listing possible tracers explicitly instead of relying on a function, shouldn't we add the remaining options? (i.e FBB and AV1451?)

Reply:
No idea if that would introduce unintended side-effects to the ADNI or AIBL converters. I'd be in favour to implement such change in a larger refactoring effort of the converters rather than taking a risk on the current implementation, which has been prone to regressions in the past.

Reply:
Ok fair enough. This will inevitably be refactored in a BIDS abstraction effort.